static/app/utils/project/useAllProjectVisibility.spec.tsx

```diff
@@ -0,0 +1,102 @@
+import type {ReactNode} from 'react';
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {ProjectFixture} from 'sentry-fixture/project';
+
+import {makeTestQueryClient} from 'sentry-test/queryClient';
+import {reactHooks} from 'sentry-test/reactTestingLibrary';
+
+import projectToProjectVisibility from 'sentry/utils/project/projectToProjectVisibility';
+import useAllProjectVisibility from 'sentry/utils/project/useAllProjectVisibility';
+import type {QueryClient} from 'sentry/utils/queryClient';
+import {QueryClientProvider} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import {getPaginationPageLink} from 'sentry/views/organizationStats/utils';
+
+jest.mock('sentry/utils/useOrganization');
+
+function makeWrapper(queryClient: QueryClient) {
+  return function wrapper({children}: {children?: ReactNode}) {
+    return <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>;
+  };
+}
+
+describe('useAllProjectVisibility', () => {
+  const mockOrg = OrganizationFixture();
+  jest.mocked(useOrganization).mockReturnValue(mockOrg);
+
+  const MOCK_API_ENDPOINT = `/organizations/${mockOrg.slug}/projects/`;
+
+  const project10 = ProjectFixture({id: '10', slug: 'ten'});
```

Comment:
might want to make a ProjectVisibilityFixture that just calls projectToProjectVisibility for you
```suggestion
  const project10 = ProjectFixture({id: '10', slug: 'ten'});
```